core/src/macros.rs

```diff
@@ -1,4 +1,3 @@
-#[macro_export]
```

Comment:
It's this macro_export which makes the macro part of the public interface I believe.